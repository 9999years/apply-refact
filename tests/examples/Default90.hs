used = not . not . any (`notElem` special) . fst . derives