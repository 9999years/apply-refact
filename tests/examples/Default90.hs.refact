[("tests/examples/Default90.hs:1:8-58: Warning: Redundant not\nFound:\n  not . not . any (`notElem` special) . fst . derives\nPerhaps:\n  any (`notElem` special) . fst . derives\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 8, endLine = 1, endCol = 59}, subts = [("x",SrcSpan {startLine = 1, startCol = 20, endLine = 1, endCol = 59})], orig = "x"}])]
