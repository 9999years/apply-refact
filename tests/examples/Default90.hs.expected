used = any (`notElem` special) . fst . derives
